source/_data/themes/travel.yml

```
@@ -0,0 +1,10 @@
+description: Travel 是一个简洁、美观且响应式的作品集主题，适用于 Hexo 静态网站生成器。
```

```suggestion
description: Travel 是一个简洁、美观且响应式的作品集主题。
```

Personally, I dont think you need to emphasize this in hexo.io